### PR TITLE
[MRG+1] Update two_scales.py example.

### PR DESCRIPTION
**Pull request by anntzer**
The docstring refers to the non-existing `axes.py`, and mentions having
to remove the rectangular frame, which is no longer the case.  I also
suspect that it was written when `twiny` didn't exist yet...

Use `tick_params` instead of iterating over the ticks.

### CODE REVIEW

File: examples/api/two_scales.py [hunk 1]
```diff
@@ -32,15 +27,14 @@
 ax1.plot(t, s1, 'b-')
 ax1.set_xlabel('time (s)')
 # Make the y-axis label and tick labels match the line color.
-ax1.set_ylabel('exp', color='b')
-for tl in ax1.get_yticklabels():
-    tl.set_color('b')
-
+ax1.set_ylabel('exp', color='r')
```
Comment (tacaswell) [inline examples/api/two_scales.py]: should this be `'b'` to match the ticks?

Comment (anntzer) [inline examples/api/two_scales.py]: oops, fixed.

File: examples/api/two_scales.py [hunk 2]
```diff
@@ -33,14 +28,13 @@
 ax1.set_xlabel('time (s)')
 # Make the y-axis label and tick labels match the line color.
 ax1.set_ylabel('exp', color='b')
-for tl in ax1.get_yticklabels():
-    tl.set_color('b')
-
+ax1.tick_params('y', color='b', labelcolor='b')
```
Comment (Kojoley) [inline examples/api/two_scales.py]: Could not this be `tick_params('y', colors='b')`?

Comment (anntzer) [inline examples/api/two_scales.py]: I didn't know that kwarg... fixed.

**Comment by NelleV** [comment]
Thanks @anntzer for the improvements!

**Comment by tacaswell** [comment]
Thanks!

Backported to v2.x as eace7a6

**Pull request merged**